src/Nancy/Json/Simple/SimpleJson.cs

```diff
@@ -1237,6 +1238,25 @@ interface IJsonSerializerStrategy
         string MapDictionaryKeyToFieldName(string stringKey);
     }
 
+#if SIMPLE_JSON_INTERNAL

```

Comment:
missing `[GeneratedCode("simple-json", "1.0.0")]` like the other internal/public classes